demo/api/src/products/products.module.ts

```diff
@@ -50,6 +51,7 @@ import { ProductVariantsService } from "./generated/product-variants.service";
         ProductToTagResolver,
         ProductColorResolver,
         CustomProductResolver,
+        ProductPublishedMail,
```

Comment:
Why do we need to provide this?

Reply:
alright, seems not to be required 👍

Reply:
I was to eager. it's required for the discovery service to find it.